src/frontend/apps/impress/src/features/docs/docs-grid/components/DocsGrid.tsx

```diff
@@ -1,220 +1,102 @@
-        );
-      },
-    },
-    {
-      headerName: t('Members'),
-      id: 'users_number',
-      renderCell: ({ row }) => {
-        return (
-          <StyledLink href={`/docs/${row.id}`}>
-            <Text $weight="bold">{row.accesses.length}</Text>
-          </StyledLink>
-        );
-      },
-    },
-    {
-      id: 'column-actions',
-      renderCell: ({ row }) => {
-        return <DocsGridActions doc={row} />;
-      },
-    },
-  ];
-
-  // Inverse columns for mobile to have the most important information first
-  if (isMobile) {
-    const tmpCol = columns[0];
-    columns[0] = columns[1];
-    columns[1] = tmpCol;
-  }
+    void fetchNextPage();
+  };
 
   return (
     <Card
-      $padding={{ bottom: 'small', horizontal: 'big' }}
-      $margin={{ all: isMobile ? 'small' : 'big', top: 'none' }}
-      $overflow="auto"
-      aria-label={t(`Datagrid of the documents page {{page}}`, { page })}
-      $height="100%"
+      data-testid="docs-grid"
+      $padding="md"
+      $width="960px"
+      $css={css`
+        width: ${isDesktop ? '960px' : '100%'};
+        max-width: calc(100vw - ${isDesktop ? LEFT_PANEL_WIDTH : 0}px - 40px);
+      `}
     >
-      <DocsGridStyle />
       <Text
-        $weight="bold"
-        as="h2"
-        $theme="primary"
-        $margin={{ bottom: 'small' }}
+        as="h4"
+        $size="h4"
+        $weight="700"
+        $margin={{ top: '0px', bottom: 'xs' }}
       >
-        {t('Documents')}
+        {t('All docs')}
       </Text>
 
-      {error && <TextErrors causes={error.cause} />}
-
-      <DataGrid
-        columns={columns}
-        rows={docs}
-        isLoading={isLoading}
-        pagination={pagination}
-        onSortModelChange={setSortModel}
-        sortModel={sortModel}
-        emptyPlaceholderLabel={t("You don't have any document yet.")}
-      />
+      <Box>
+        <Box $direction="row" $padding="xs" data-testid="docs-grid-header">
+          <Box $flex={6} $padding="3xs">
+            <Text $size="xs" $variation="600">
+              {t('Name')}
+            </Text>
+          </Box>
+          {isDesktop && (
+            <Box $flex={1} $padding="3xs">
+              <Text $size="xs" $variation="600">
+                {t('Updated at')}
+              </Text>
+            </Box>
+          )}
+
+          <Box $flex={1} $align="flex-end" $padding="3xs" />
+        </Box>
+        {/* Body */}
+        {data?.pages.map((currentPage) => {
+          return currentPage.results.map((doc) => (
+            <DocsGridItem doc={doc} key={doc.id} />
+          ));
+        })}
+      </Box>
+
+      {loading && (
+        <Box
+          data-testid="docs-grid-loader"
+          $padding="md"
+          $align="center"
+          $justify="center"
+          $width="100%"
+        >
+          <Loader />
+        </Box>
+      )}
+      {hasNextPage && (
+        <InView
+          data-testid="infinite-scroll-trigger"
+          as="div"
+          onChange={loadMore}
```

Comment:
It seems to be triggered twice:
![image](https://github.com/user-attachments/assets/9e298b49-aa9e-4d61-9203-a675c6d76626)